### DISCUSSION TITLE
We're still processing your application. Once this is complete you will have access to the student developer pack offers.

### DISCUSSION THREAD

**Discussion by BreadCatto** [community/community#72928] [GitHub Education]
### Hi everyone,

Its been 3 days but still im getting this
![image](https://github.com/community/community/assets/90331269/00308e36-1202-4bbe-872a-1299b5919832)
![image](https://github.com/community/community/assets/90331269/e0215ed1-ace2-4a2c-8c74-d882f068afb5)

**Comment by Sabrina-Lopez** [comment]
Yea, I have waited 2 weeks to receive the GitHub Student Developer Pack benefits and offers, and I have an approved status for my application. I would like to at least know the answer to this problem too.
